ci/generate_changelog.py

```diff
@@ -0,0 +1,203 @@
+"""OS module for file operations,
```

Comment:
Nice script, did you run black on it ?

Reply:
Yeah, I ran both black and pylint manually, but only now noticed that the ci/ dir is not being checked by our CI linters, so I'll create a ticket for it